package.json

```diff
@@ -9,10 +9,13 @@
     "lint": "bunx --bun next lint"
   },
   "dependencies": {
+    "@dimidumo/zk-email-sdk-ts": "^1.0.13",
+    "copy-webpack-plugin": "^12.0.2",
```

Comment:
Where are we using this and why?

Reply:
That was to make wasm work for node, which didn't work, forgot to remove it.